pkg/component/kubeproxy/resources.go

```diff
@@ -559,6 +466,11 @@ func (k *kubeProxy) computePoolResourcesData(pool WorkerPool) (map[string][]byte
 		daemonSet.Spec.Template.Spec.Containers[0].Resources.Limits = corev1.ResourceList{
 			corev1.ResourceMemory: resource.MustParse("2048Mi"),
 		}
+		if k8sGreaterEqual129 {
+			daemonSet.Spec.Template.Spec.InitContainers[1].Resources.Limits = corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("2048Mi"),
```

Comment:
IMO, there is no need for such high memory limit. AFAIK it should be a only an init container that sets kernel parameters and exits:
```suggestion
				corev1.ResourceMemory: resource.MustParse("256Mi"),
```